### ISSUE TITLE
Walkthrough of the Azure workflow

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/pyrenew-hew#82]
> Is there any readme explaining what each one of the demos do?

_Originally posted by @gvegayon in https://github.com/CDCgov/pyrenew-hew/pull/78#discussion_r1819810347_

**Comment by damonbayer** [comment]
I don't think this is worth documenting, as we plan to move to some workflow orchestration tool in the near future.
